src/sql/workbench/contrib/executionPlan/browser/media/executionPlan.css

```diff
@@ -110,18 +110,18 @@ However we always want it to be the width of the container it is resizing.
 }
 
 /* Properties view in execution plan */
-.eps-container .execution-plan .properties {
+.properties {
 	flex: 0 0 500px;
 	overflow: hidden;
 	width: 500px;
-	height: calc( 100% - 2px );
+	height: calc(100% - 2px);
 	display: flex;
 	flex-direction: column;
 	border-left: 1px solid;
 }
 
 /* Title container of the properties view */
-.eps-container .execution-plan .properties .title {
+.properties .title {
```

Comment:
Everything in this file needs to be scoped down to at least the eps-container level